.github/workflows/build.yml

```diff
@@ -12,9 +18,6 @@ jobs:
     runs-on: ubuntu-latest
     continue-on-error: true
     name: Build on Raspberry Pi ${{ matrix.arch }}
-    env:
-      GITHUB_ENV: CLOUDSMITH_API_KEY
-      CLOUDSMITH_API_KEY: ${{ secrets.CLOUDSMITH_KEY }}
```

Comment:
At least this line needs to stay, it makes the API key available in the container

Reply:
the line was moved to the 'upload' job